app/models/carto/user.rb

```diff
@@ -222,15 +225,48 @@ def google_maps_client_id
   end
 
   # returns a list of basemaps enabled for the user
+  # when google map key is set it gets the basemaps inside the group "GMaps"
+  # if not it get everything else but GMaps in any case GMaps and other groups can work together
+  # this may have change in the future but in any case this method provides a way to abstract what
+  # basemaps are active for the user
+  #
+  # Basemaps may also be filtered by profile attributes - this implementation respects such filters.
   def basemaps
-    (Cartodb.config[:basemaps] || []).select { |group| group != 'GMaps' || google_maps_enabled? }
```

Comment:
Although we don't enable google maps this will lead to merge conflicts later.  Is there an issue with simply keeping google_maps_enabled to false?

Reply:
Logic kept and moved to the end of the function.

Reply:
The logic appears to have moved to a new function filter_basemaps rather than the stay in the pre-existing base maps function.  Functionally it's moot since Bloomberg doesn't use google maps base layers.  I'm simply mentioning it for merging upstream carto code in the future.

Reply:
`filter_basemaps` is a helper function within the scope of `basemaps` - the google maps logic still appears in the last line of `basemaps`.